### ISSUE TITLE
Timeseries add-on gets installed but does not appear in the add-on list

### ISSUE THREAD

**Issue by mah12man** [biolab/orange3-timeseries#191]
I tried installing the 'timeseries add-on' through all means - the add-on installer, pip and conda, but it won't appear in the toolbox. The installation is successful every time, without any errors, but there's no sign of this add-on. The installer however indicates that it has been installed.  

##### Timeseries version

0.3.12

##### Orange version

3.32.0

##### Expected behavior

Add-on should appear after restart

##### Actual behavior

Add-on not present in toolbox

**Comment by layudhi** [comment]
I have same issue, Time Series Widget was installed successfully but does not appear in the toolbox.
I tried to uninstall and re-installed again, but the problem persist 

- Run on Windows 7 64 Bit
- Orange 3.32.0

![Preview](https://user-images.githubusercontent.com/102269946/164979945-014be6f7-fc65-4d90-8127-77b471d1f41e.png)

**Comment by kernc** [comment]
Likely a duplicate of https://github.com/biolab/orange3-timeseries/issues/188.

**Comment by mah12man** [comment]
Yes. It's a duplicate of #188. Thanks for letting me know of this thread. The issue was resolved using the solution provided. Scipy needs to be replaced by an older version.
